_posts/2024-05-03-trino-snowflake-bloomberg-oss-win.md

```diff
@@ -0,0 +1,294 @@
+---
+layout: post
+title:  "Integrating Trino and Snowflake: An open source success story"
+author: Brian Olsen
+excerpt_separator: <!--more-->
+canonical_url: https://bitsondata.dev/trino-snowflake-bloomberg-oss-win
+---
+
+We’re seeing open source usher in a challenge to the economic model where the 
+success metric is increasing the commonwealth of economic capital. This 
+acceleration comes from playing positive-sum games with friends online and 
+avoiding limiting a community to a vision that only benefits a small number of 
+corporations or individuals. It’s hard to imagine how to embed such frameworks 
+within our current zero-sum winner-takes-all economic system. There’s certainly 
```

Comment:
Brian Olsen takes down capitalism, one open source blog post at a time (I do like that you're not mincing words here)

Reply:
haha .. currently reading https://en.wikipedia.org/wiki/Chokepoint_Capitalism myself ..